cli/config/configStructs/tapConfig.go

```diff
@@ -71,9 +77,48 @@ func (config *TapConfig) GetInsertionFilter() string {
 			}
 		}
 	}
+
+	redactFilter := getRedactFilter(config)
+	if insertionFilter != "" && redactFilter != "" {
+		return fmt.Sprintf("(%s) and %s", insertionFilter, redactFilter)
```

Comment:
```suggestion
		return fmt.Sprintf("(%s) and (%s)", insertionFilter, redactFilter)
```

Reply:
redact filter is only one command - redact(<list_of_values_to_redact>)
so i'm not sure it's necessary